AutoCrop3D/CMakeLists.txt

```diff
@@ -0,0 +1,3 @@
+add_subdirectory(Crop_Volumes_CLI)
+add_subdirectory(Crop_Volumes_UI)
+include(ImportLibrary.cmake)
```

Comment:
There is no `ImportLibrary.cmake` present to be included. You must create this file and declare the contents of `Crop_Volumes_utils`, or else these files will not be available in the extension when installed via the Extension Manager in Slicer.

As reference, see the `ImportLibrary.cmake` for ASO_CBCT. It declares the contents of ASO_CBCT_utils so they are available for use in the CLI module.

https://github.com/DCBIA-OrthoLab/SlicerAutomatedDentalTools/tree/main/ASO_CBCT

Reply:
This should also work fine; the important thing is to be sure those files are declared _somewhere_ so that they are installed for users via the extension manager.